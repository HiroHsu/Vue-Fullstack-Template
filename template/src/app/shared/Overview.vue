<template>
  <div>
    <v-jumbotron dark gradient="to bottom, #1E88E5, #42A5F5" height="auto">
      <v-container fill-height>
        <v-layout justify-center align-center wrap>
          <img src="assets/images/logo.svg" alt="Logo" width="300" height="300" />

          <v-flex shrink>
            <h3 class="display-3">{{ name }}</h3>
            <span class="subheading">{{ description }}</span>
            <v-divider class="my-3"></v-divider>
            <v-btn large color="white" to="/hello-world" class="primary--text ml-0">See more</v-btn>
            <v-btn color="white" href="https://github.com/Shyam-Chen/Vue-Fullstack-Starter" large outline target="_blank">
              <v-icon left>fa fa-github</v-icon>
              GitHub
            </v-btn>
            <v-btn color="white" href="https://github.com/Shyam-Chen/Vue-Fullstack-Template" large outline target="_blank">
              <v-icon left>fa fa-file-code-o</v-icon>
              Template
            </v-btn>
          </v-flex>
        </v-layout>
      </v-container>
    </v-jumbotron>

    <v-container grid-list-xl class="mt-5 mb-5">
      <v-layout row justify-center wrap>
        <v-flex xs12 sm8 md4>
          <div class="elevation-1" style="height: 256px;">

          </div>
        </v-flex>

        <v-flex xs12 sm8 md4>
          <div class="elevation-1" style="height: 256px;">

          </div>
        </v-flex>
      </v-layout>
    </v-container>

    <v-container grid-list-xl class="mt-5 mb-5">
      <v-layout row justify-center wrap>
        <v-flex xs12 sm8 md4>
          <div class="elevation-1" style="height: 256px;">

          </div>
        </v-flex>

        <v-flex xs12 sm8 md4>
          <div class="elevation-1" style="height: 256px;">

          </div>
        </v-flex>
      </v-layout>
    </v-container>

    <v-container grid-list-xl class="mt-5 mb-5">
      <v-layout row justify-center wrap>
        <v-flex xs12 sm8 md4>
          <div class="elevation-1" style="height: 256px;">

          </div>
        </v-flex>

        <v-flex xs12 sm8 md4>
          <div class="elevation-1" style="height: 256px;">

          </div>
        </v-flex>
      </v-layout>
    </v-container>

    <v-container grid-list-xl class="mt-5 mb-5">
      <v-layout row justify-center>
        <v-btn large color="primary" to="/hello-world">See more</v-btn>
      </v-layout>
    </v-container>
  </div>
</template>

<script>
export default {
  metaInfo: {
    title: '{{ name }}',
  },
};
</script>

<style scoped>
</style>
