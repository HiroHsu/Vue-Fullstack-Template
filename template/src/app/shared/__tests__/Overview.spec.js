import { shallow } from '@vue/test-utils';

import Overview from '../Overview';

describe('Overview', () => {
  let [wrapper] = [];

  beforeEach(() => {
    wrapper = shallow(Overview);
  });

  it('should render initial component', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
