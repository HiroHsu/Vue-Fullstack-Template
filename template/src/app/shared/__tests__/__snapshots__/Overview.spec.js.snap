// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Overview should render initial component 1`] = `
<div>
  <v-jumbotron dark="" gradient="to bottom, #1E88E5, #42A5F5" height="auto">
    <v-container fill-height="">
      <v-layout justify-center="" align-center="" wrap="">
        <img src="assets/images/logo.svg" alt="Logo" width="300" height="300">
        <v-flex shrink="">
          <h3 class="display-3">Vue Fullstack</h3> <span class="subheading">Vue, Material, Express, Babel, Flow, and ReactiveX.</span>
          <v-divider class="my-3"></v-divider>
          <v-btn large="" color="white" to="/hello-world" class="primary--text ml-0">See more</v-btn>
          <v-btn color="white" href="https://github.com/Shyam-Chen/Vue-Fullstack-Starter" large="" outline="" target="_blank">
            <v-icon left="">fa fa-github</v-icon>
            GitHub
          </v-btn>
          <v-btn color="white" href="https://github.com/Shyam-Chen/Vue-Fullstack-Template" large="" outline="" target="_blank">
            <v-icon left="">fa fa-file-code-o</v-icon>
            Template
          </v-btn>
        </v-flex>
      </v-layout>
    </v-container>
  </v-jumbotron>
  <v-container grid-list-xl="" class="mt-5 mb-5">
    <v-layout row="" justify-center="" wrap="">
      <v-flex xs12="" sm8="" md4="">
        <div class="elevation-1" style="height: 256px;">
          Standard Image
        </div>
      </v-flex>
      <v-flex xs12="" sm8="" md4="">
        <div class="elevation-1" style="height: 256px;">
          Standard Content
        </div>
      </v-flex>
    </v-layout>
  </v-container>
  <v-container grid-list-xl="" class="mt-5 mb-5">
    <v-layout row="" justify-center="" wrap="">
      <v-flex xs12="" sm8="" md4="">
        <div class="elevation-1" style="height: 256px;">
          Playground Content
        </div>
      </v-flex>
      <v-flex xs12="" sm8="" md4="">
        <div class="elevation-1" style="height: 256px;">
          Playground Image
        </div>
      </v-flex>
    </v-layout>
  </v-container>
  <v-container grid-list-xl="" class="mt-5 mb-5">
    <v-layout row="" justify-center="" wrap="">
      <v-flex xs12="" sm8="" md4="">
        <div class="elevation-1" style="height: 256px;">
          Profound Image
        </div>
      </v-flex>
      <v-flex xs12="" sm8="" md4="">
        <div class="elevation-1" style="height: 256px;">
          Profound Content
        </div>
      </v-flex>
    </v-layout>
  </v-container>
  <v-container grid-list-xl="" class="mt-5 mb-5">
    <v-layout row="" justify-center="">
      <v-btn large="" color="primary" to="/hello-world">See more</v-btn>
    </v-layout>
  </v-container>
</div>
`;
